### ISSUE TITLE
Advice wanted: iframe fallback to img

### ISSUE THREAD

**Issue by vibrog** [robflaherty/html-slideshow#6]
Consider wanting to use iframes in a presentation relying on having an Internet connection, but wanting to be able to use "cached" screenshots if in an offline setting.

```
<iframe src="http://example.com">
  <img src="screenshot-example.com.png"/>
</iframe>
```

How can that be solved in a neat way?

**Comment by robflaherty** [comment]
Did you ever find a solution for this?  What about adding a "offline" setting that uses JavaScript to remove the iframes wrapping your fallback content?

**Comment by vibrog** [comment]
No. My initial thought was to rely on the `<object>`-element's alternative/hierarchical nature.
